### ISSUE TITLE
Effective sample size is applied even if not multinomial

### ISSUE THREAD

**Issue by cgrandin** [smartell/iSCAM#52]
Not sure if this was by design, this happens in the data file loading part of the tpl. Seems that the code should check if multinomial here:

			if(inp_nscaler(k) > 0)
				{
					for(i = 1; i <= n_A_nobs(k); i++ )
					{
						 tmp(i) = tmp(i)/sum(tmp(i)) * inp_nscaler(k);
					}
				}

**Comment by smartell** [comment]
Good point.  I suppose the conditional statement should have 2 conditions

**Comment by cgrandin** [comment]
That's what I thought. I just sent you an email as well thinking you weren't looking on github.

**Comment by smartell** [comment]
The intention was to be able to scale the age-comp proportions to some other sample size when the data were entered in as proportions.
